src/Infrastructure/BucketTesting/BucketSelectionServiceProvider.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare( strict_types = 1 );
+
+
+namespace WMDE\Fundraising\Frontend\Infrastructure\BucketTesting;
+
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Silex\Api\BootableProviderInterface;
+use Silex\Application;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use WMDE\Fundraising\Frontend\Factories\FunFunFactory;
+
+class BucketSelectionServiceProvider  implements ServiceProviderInterface, BootableProviderInterface {
+
+	private const COOKIE_NAME = 'spenden_ttg';
+	private const COOKIE_LIFETIME_IN_SECONDS = 60 * 60 * 24 * 90; // 90 days
```

Comment:
Three months was only an assumption. See https://phabricator.wikimedia.org/T196335#4301120 for necessary changes.